hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerPrepare.java

```diff
@@ -296,7 +297,7 @@ public void testPrepareWithMultipleThreads() throws Exception {
               .anyMatch((vol) -> vol.getName().equals(volumeName)));
         } catch (ExecutionException ex) {
           Throwable cause = ex.getCause();
-          assertTrue(cause instanceof OMException);
+          assertInstanceOf(OMException.class, cause);
           assertEquals(
               NOT_SUPPORTED_OPERATION_WHEN_PREPARED,
               ((OMException) cause).getResult());
```

Comment:
We can simplify this as:

```java
OMException cause = assertInstanceOf(OMException.class, cause);
assertEquals(NOT_SUPPORTED_OPERATION_WHEN_PREPARED, cause.getResult());
```

Reply:
Thanks for review. fixed comments,please take a look when you have time.